README.md

```diff
@@ -0,0 +1,22 @@
+[![Build Status](https://travis-ci.org/syslabcom/slc.outdated.svg?branch=master)](https://travis-ci.org/syslabcom/slc.outdated)
```

Comment:
To have proper markdown support for use package use this https://github.com/collective/zpretty/blob/master/setup.py#L18, remove `README.txt` and replace it with `README.md` in the `setup.py`

Reply:
Thanks, good to know!